tests/robustness/engine/engine.go

```diff
@@ -213,55 +230,56 @@ func (e *Engine) Init(ctx context.Context, testRepoPath, metaRepoPath string) er
 	engineMode := os.Getenv(EngineModeEnvKey)
 
 	switch {
-	case bucketName != "" && engineMode == EngineModeBasic:
-		return e.InitS3(ctx, bucketName, testRepoPath, metaRepoPath)
+	// case bucketName != "" && engineMode == EngineModeBasic:
+	// 	return e.InitS3(ctx, bucketName, testRepoPath, metaRepoPath)
 
 	case bucketName != "" && engineMode == EngineModeServer:
 		return e.InitS3WithServer(ctx, bucketName, testRepoPath, metaRepoPath, defaultAddr)
 
 	case bucketName == "" && engineMode == EngineModeServer:
 		return e.InitFilesystemWithServer(ctx, testRepoPath, metaRepoPath, defaultAddr)
 
-	default:
-		return e.InitFilesystem(ctx, testRepoPath, metaRepoPath)
+		// default:
+		// 	return e.InitFilesystem(ctx, testRepoPath, metaRepoPath)
 	}
+	return nil
 }
 
 // InitS3 attempts to connect to a test repo and metadata repo on S3. If connection
 // is successful, the engine is populated with the metadata associated with the
 // snapshot in that repo. A new repo will be created if one does not already
 // exist.
-func (e *Engine) InitS3(ctx context.Context, bucketName, testRepoPath, metaRepoPath string) error {
-	err := e.MetaStore.ConnectOrCreateS3(bucketName, metaRepoPath)
-	if err != nil {
-		return err
-	}
+// func (e *Engine) InitS3(ctx context.Context, bucketName, testRepoPath, metaRepoPath string) error {
```

Comment:
`InitS3` would not be used for multiple clients and server model, so just commented it out for clearity.